Makefile.common

```diff
@@ -131,5 +137,5 @@ $(PREFIX_O)%.o: %.S
 	
 $(PREFIX_PROG_STRIPPED)%: $(PREFIX_PROG)%
 	@mkdir -p $(@D)
-	@(printf "STR %-24s\n" "$(@F)")
+	@(printf "STR %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)")
```

Comment:
```suggestion
	@printf "STR %-24s\n" "$(@:$(PREFIX_BUILD)/%=%)"
```
as above

Reply:
not related to this PR, so leaving as-is. I agree that the parentheses are not needed in this case.